admin_guide/node-allocatable-resources.adoc

```diff
@@ -0,0 +1,82 @@
+= Node Allocatable resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node has a chance to reserve a portion of its resources for use by all
+openshift-node underlying components (e.g. kubelet, kube-proxy, docker)
+and remaining system components (e.g. sshd, NetworkManager) on a node.
+Once specified, scheduler has more information about how much resources
+(e.g. memory, cpu) a node has allocated for pods.
+
+== Settings
+
+Resources reserved for components are based on two node settings:
+
+[options="header",cols="1,2"]
+|===
+
+|Setting |Description
+
+|`*kube-reserved*`
+| A set of `ResourceName=ResourceQuantity` (e.g. cpu=200m,memory=150G) pairs that describe resources reserved for openshift-node components. Default is none.
```

Comment:
reserving 200m of cpu is sensible, 150G of memory is not.  maybe reduce to a smaller number to avoid making us look like a memory hog or having confusion.